library/globals.inc.php

```diff
@@ -231,11 +231,11 @@ function gblTimeZones()
             xl('Default font size') . '*',
             array(
                 '__default__' => 'Use Theme Font Size',
-                '0.625rem' => '10px',
-                '0.75rem' => '12px',
-                '0.875rem' => '14px',
-                '1rem' => '16px',
-                '1.125rem' => '18px',
+                '0.625' => '10px',
+                '0.75' => '12px',
+                '0.875' => '14px',
+                '1.0' => '16px',
+                '1.125' => '18px',
```

Comment:
TODO for me:
Gonna need some stuff in sql upgrade to fix this for upgraders

Reply:
this should go into sql/5_0_2-to-6_0_0_upgrade.sql:
```
UPDATE `globals` SET `gl_value`='0.625' WHERE `gl_name`='font-size' AND `gl_value`='0.625rem';
UPDATE `globals` SET `gl_value`='0.75' WHERE `gl_name`='font-size' AND `gl_value`='0.75rem';
UPDATE `globals` SET `gl_value`='0.875' WHERE `gl_name`='font-size' AND `gl_value`='0.875rem';
UPDATE `globals` SET `gl_value`='1.0' WHERE `gl_name`='font-size' AND `gl_value`='1rem';
UPDATE `globals` SET `gl_value`='1.125' WHERE `gl_name`='font-size' AND `gl_value`='1.125rem';
```

Reply:
Done